runtime/gc_base/GCExtensions.cpp

```diff
@@ -85,32 +85,6 @@ MM_GCExtensions::initialize(MM_EnvironmentBase *env)
 		goto failed;
 	}
 
-#if defined(OMR_ENV_DATA64)
-	if (J2SE_VERSION((J9JavaVM *)getOmrVM()->_language_vm) >= J2SE_19) {
-		uint64_t physicalMemory = 0;
-		uint64_t memoryLimit = 0;
-		uint64_t usableMemory = 0;
-		/* Initial physicalMemory as per system call. */
-		physicalMemory = j9sysinfo_get_physical_memory();
-		if (OMRPORT_LIMIT_LIMITED == j9sysinfo_get_limit(OMRPORT_RESOURCE_ADDRESS_SPACE, &memoryLimit)) {
-			/* there is a limit on the memory we can use so take the minimum of this usable amount and the physical memory */
-			usableMemory = OMR_MIN(memoryLimit, physicalMemory);
-		} else {
-			/* if there is no memory limit being imposed on us, we will use physical memory as our max */
-			usableMemory = physicalMemory;
-		}
-
-		/* extend java default max memory to 25% of physical RAM */
-		memoryMax = OMR_MAX(memoryMax, usableMemory / 4);
-		/* limit maxheapsize up to MAXIMUM_HEAP_SIZE_RECOMMENDED_FOR_3BIT_SHIFT_COMPRESSEDREFS, then can set 3bit compressedrefs as the default */
-		memoryMax = OMR_MIN(memoryMax, MAXIMUM_HEAP_SIZE_RECOMMENDED_FOR_3BIT_SHIFT_COMPRESSEDREFS);
-
-		/* Initialize Xmx, Xmdx */
-		memoryMax = MM_Math::roundToFloor(heapAlignment, (uintptr_t)memoryMax);
-		maxSizeDefaultMemorySpace = memoryMax;
```

Comment:
Was this line lost in the code refactoring?

Reply:
Yup, thanks for catching it.